### PR TITLE
9 header component [WIP]

### PR DESCRIPTION
**Pull request by kaczynska**

### CODE REVIEW

File: frontend/src/components/Header.vue [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+.active {
+  opacity: .8;
+}
+.v-btn {
+  font-weight: 700;
+}
+.v-menu {
+  margin-right: 20px;
+}
+</style>
```
Comment (w1stler) [inline frontend/src/components/Header.vue]: No newline

**Comment by w1stler** [comment]
Do we want to have logo in header instead of plain text? Currently it's this way. We should also remove the white box beneath the header.

**Comment by kaczynska** [comment]
> Do we want to have logo in header instead of plain text? Currently it's this way. We should also remove the white box beneath the header.
> 
> https://ibb.co/0265wWP
> https://ibb.co/RCxjcY1

We want to use the svg format because it is optimal and completely compatible with the original version of the character. When using text, we do not have control over the distance between letters, and these in practice are often adjusted manually (at the stage of working on the sign).

The white box should of course be removed.

File: frontend/src/assets/logo-white.svg [hunk 2]
```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (magul) [inline frontend/src/assets/logo-white.svg]: if we're going to have logos in SVG can we also store them in https://github.com/CodeForPoznan/Community/tree/master/logos

**Comment by w1stler** [comment]
Closing - @stanislawK is working on this in #227

**Pull request closed without merging**